opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -672,16 +672,64 @@ bool FieldProps::supported<int>(const std::string& keyword) {
     return Fieldprops::keywords::isFipxxx(keyword);
 }
 
+void FieldProps::apply_multipliers()
+{
+    static const auto prefix = FieldPropsManager::getMultiplierPrefix();
+
+    for(auto pos =multiplier_kw_infos_.begin(); pos != multiplier_kw_infos_.end(); ++pos)
+    {
+        const auto& [mult_keyword, kw_info] = *pos;
```

Comment:
Is there a reason for not using a range-for of the form
```C++
for (const auto& [mult_keyword, kw_info] : multiplier_kw_infos_) {
    ...
}
```
instead?

Reply:
Historic baggage from the unexpectedly long journey to the solution. :sweat_smile: Changed